src/api/utils/descriptions.ts

```diff
@@ -4,10 +4,19 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
+import { ApiForbiddenResponse, ApiNotFoundResponse, ApiUnauthorizedResponse, } from "@nestjs/swagger";
+
 export const unauthorizedDescription =
   'Authorization information is missing or invalid';
 export const forbiddenDescription =
   'Access to the requested resource is not permitted';
 export const notFoundDescription = 'The requested resource was not found';
 export const successfullyDeletedDescription =
   'The requested resource was sucessfully deleted';
+
+export const FullApi: MethodDecorator & ClassDecorator = function FullApi(target: any) {
```

Comment:
Please define a function you want to export:

```suggestion
export function FullApi(target: any): : MethodDecorator & ClassDecorator {
```